docs/guides/expectations/create_expectations_overview.md

```diff
@@ -3,4 +3,67 @@ title: "Create Expectations: Overview"
 ---
 # [![Create Expectations Icon](../../images/universal_map/Flask-active.png)](./create_expectations_overview.md) Create Expectations: Overview
 
-** This is a placeholder until content is created, so that corresponding links don't cause docusaurus to break **
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+
+<!--Use 'inactive' or 'active' to indicate which Universal Map steps this term has a use case within.-->
+
+<UniversalMap setup='inactive' connect='inactive' create='active' validate='inactive'/>
+
+:::note Prerequisites
+- Completing [Step 3: Create Expectations](/docs/tutorials/getting_started/create_your_first_expectations) of the Getting Started tutorial is recommended.
+:::
+
+Creating Expectations is an integral part of Great Expectations.  By the end of this step, you will have created an Expectation Suite containing one or more Expectations which you will use when you Validate data.
+
+## The Create Expectations process
```

Comment:
Love this introductory section! Could we add some bolding or other formatting to key terms (especially terms like "interactive workflow")? It might help break up the section a bit.

Reply:
I added bold to each of the three types of workflows, so one bold statement near the start of each paragraph to highlight what that paragraph would be about.